README.md

```diff
@@ -52,7 +52,7 @@ Once Chrome and Java are installed, use your file manager to navigate to the sco
 
 * Apple users: Run scoreboard-Mac.app to start the script. If you are unable to start it, you may have to download and run the [startscoreboard-mac.zip package](https://drive.google.com/file/d/0B2fZmT3bqA9oS25KNHphZjFFcWs/view?fref=gc&dti=480408282040949) to get the scoreboard started.
 
-* Linux users: Run scoreboard-Linux.sh to start the script. If you are unable to start it, you may have to allow script files to be executable as programs.
+* Linux users: Run scoreboard-Linux.sh to start the script. If you are unable to start it, you may have to allow script files to be executable as programs. The ``lib/crg-scoreboard.jar`` file can be built from source by installing ``apache-ant`` and running ``ant compile`` from this directory.
```

Comment:
On Ubuntu the package is `ant`, it's probably best to just link to http://ant.apache.org/